### PR TITLE
fix: fix first failing tx for staking

### PR DESCRIPTION
**Pull request by nduchak**

### CODE REVIEW

File: src/components/organisms/storage/staking/Staking.tsx [hunk 1]
```diff
@@ -138,7 +138,14 @@ const Staking: FC = () => {
...
       const stakeContract = StakingContract.getInstance(web3 as Web3)
       const receipt = await stakeContract.stake(
-        amount, { token: currency, from: account },
+        amount,
+        {
+          token: currency,
+          from: account,
+          ...parseFloat(totalStakedUSD) > 0
+            ? {}
+            : { gasMultiplier: 3 },
```
Comment (julianmrodri) [inline src/components/organisms/storage/staking/Staking.tsx:147]:
```suggestion
            : { gasMultiplier: 1.8 },
```

Comment (julianmrodri) [reply src/components/organisms/storage/staking/Staking.tsx:147]: 1.8 is enough gives me around 125K gas..

**Review by julianmrodri — DISMISSED** [review]
LGTM! Worked nice. Lets change the multiplier to smaller number.

File: src/components/organisms/storage/staking/Staking.tsx (hunk 1 above)
Comment (nduchak) [reply src/components/organisms/storage/staking/Staking.tsx]: Done

**Review by julianmrodri — APPROVED** [review]
LGTM

File: src/components/organisms/storage/staking/Staking.tsx [hunk 2]
```diff
@@ -138,7 +138,14 @@ const Staking: FC = () => {
 
       const stakeContract = StakingContract.getInstance(web3 as Web3)
       const receipt = await stakeContract.stake(
-        amount, { token: currency, from: account },
+        amount,
+        {
+          token: currency,
+          from: account,
+          ...parseFloat(totalStakedUSD) > 0
```
Comment (jurajpiar) [inline src/components/organisms/storage/staking/Staking.tsx:145]: Could we just do
```suggestion
          ...parseFloat(totalStakedUSD)
```
?

**Review by jurajpiar — APPROVED** [review]
Sweet as

File: src/components/organisms/storage/staking/Staking.tsx (hunk 2 above)
Comment (nduchak) [reply src/components/organisms/storage/staking/Staking.tsx:145]: Yes, we can. I just prefer to don't make type conversion implicitly, as you know JS is very tricky and stupid in dynamic type conversion

**Pull request merged**